services/table-foreign-keys-analyzer.js

```diff
@@ -29,8 +30,9 @@ function TableForeignKeysAnalyzer(databaseConnection, config) {
             REFERENCED_TABLE_NAME AS foreign_table_name,
             REFERENCED_COLUMN_NAME AS foreign_column_name
           FROM INFORMATION_SCHEMA.KEY_COLUMN_USAGE
-          WHERE TABLE_SCHEMA = '${config.dbName}'
-            AND TABLE_NAME = '${table}';`;
+          WHERE TABLE_SCHEMA = :databaseName
+            AND TABLE_NAME = :table;`;
+        replacements.databaseName = queryInterface.sequelize.config.database;
```

Comment:
[MISC] Much better 👍

Reply:
But when I look at the doc I see `getDatabaseName()` which seems more friendly and public to me don't you think 🤔 ?
https://sequelize.org/master/class/lib/sequelize.js~Sequelize.html

Reply:
Ahah it's sequelize....